angular-hub/src/app/pages/podcasts/index.page.ts

```diff
@@ -45,19 +46,43 @@ export default class PodcastsComponent {
     searchControl = new FormControl<string>('', {nonNullable: true});
     podcasts = injectContentFiles<Podcast>(({filename}) => filename.startsWith('/src/content/podcasts/'));
 
-    podcasts$ = this.searchControl.valueChanges.pipe(
-        startWith(''),
-        map((searchTerm) => {
+    q = signal('');
+    filterOptions = computed(() => ({searchTerm: this.q()}));
+
+    podcasts$ = toObservable(this.filterOptions).pipe(
```

Comment:
use the same `route.queryParams` strategy as the other components for both consistency and simplicity (without Signal Inputs, it adds a lot of code).